mailpoet/RoboFile.php

```diff
@@ -957,6 +957,15 @@ public function releasePublishSlack($version = null) {
     $this->say("Release '$version[name]' info was published on Slack.");
   }
 
+  public function downloadWooCommerceBlocksZip($tag = null) {
+    if (!getenv('WP_GITHUB_USERNAME') && !getenv('WP_GITHUB_TOKEN')) {
+      $this->yell("Skipping download of WooCommerce Blocks", 40, 'red');
+      exit(0); // Exit with 0 since it is a valid state for some environments
+    }
+    $this->createGithubClient('woocommerce/woocommerce-gutenberg-products-block')
+      ->downloadReleaseZip('woo-gutenberg-products-block.zip', __DIR__ . '/tests/plugins/', $tag);
+  }
+
```

Comment:
Just FYI: We added a tool for downloading free plugins directly from repo see https://github.com/mailpoet/mailpoet/blob/master/mailpoet/RoboFile.php#L978-L981
There is no need to change it. Just maybe next time, it might be easier to use and works without a token.